config/crd/bases/external-secrets.io_externalsecrets.yaml

```diff
@@ -93,11 +100,13 @@ spec:
                   type: object
                 type: array
               refreshInterval:
+                default: 1h
                 description: 'RefreshInterval is the amount of time before the values
-                  reading again from the SecretStore provider Valid time units are
-                  "ns", "us" (or "µs"), "ms", "s", "m", "h" (from time.ParseDuration)
-                  May be set to zero to fetch and create it once TODO: Default to
-                  some value?'
+                  reading again from the SecretStore provider We support a subset
+                  of time.ParseDuration: Valid time units are "ns", "us" (or "µs"),
+                  "ms", "s", "m", "h" May be set to zero to fetch and create it once.
+                  Defaults to 1h.'
```

Comment:
```yaml
                description: 'RefreshInterval is the amount of time before the values
                  are read again from the SecretStore provider. We support a subset
                  of time.ParseDuration: Valid time units are "ns", "us" (or "µs"),
                  "ms", "s", "m", "h" May be set to zero to fetch and create it once.
                  Defaults to 1h.'
```

`values reading again` -> `values are read again`

period between `(...) provider` and `We support (...)` 😄